### ISSUE TITLE
Music page doesnt render correctly on a tablet (Pinetab2)

### ISSUE THREAD

**Issue by piggz** [heartfin/harbour-sailfin#43]
On my pinetab2, i can navigate the main UI, through to Music and get the list of latest/albums.

When clicking an Album, the next page loads, however its not possible to select any tracks to play.

In portrait, the album art takes the full page and nothing else is visible.

In landscape, the album art takes the full screen again, but i can see some track lengths along the right side, but nothing else.

**Comment by HenkKalkwater** [comment]
Could you perhaps send some screenshots? I do not have a tablet and it looked fine in landscape mode on my phone.

**Comment by piggz** [comment]
![photo_2024-01-15_20-22-04](https://github.com/heartfin/harbour-sailfin/assets/4650063/56fde06b-4395-4a67-8d30-02290dbb85e7)
![photo_2024-01-15_20-21-58](https://github.com/heartfin/harbour-sailfin/assets/4650063/7838619b-a64b-4177-84e6-087db7b9cf3f)

Sorry theyre not screenshots ... the pt2 takes empty screenshots :)

**Comment by piggz** [comment]
Hi, thanks for trying, but I see no change :/

**Comment by piggz** [comment]
Hi, I took a look at this ... the issue seems to be here
        Loader {          
            id: wideAlbumCover
            visible: _twoColumns
            Layout.minimumWidth: 1000 / Theme.pixelRatio
            Layout.fillHeight: true
            source: visible
                    ? "../../components/music/WideAlbumCover.qml" : ""
            onLoaded: bindAlbum(item)
        }
in pages/itemdetails/MusicAlbumPage.qml

Setting the value from 1000 to 500 works much better here, in landscape mode on the pt2.  Maybe some value that is proportional to the screen could be found, and not hard coded?

**Comment by piggz** [comment]
In portrait mode, it still looks bad, but its this are that the issue

**Comment by HenkKalkwater** [comment]
Alright, I finally got Sailfin working in the emulator. It turned out that I needed to install the `qt5-qtmultimedia-plugin-mediaservice-gstmediacapture` and `qt5-qtmultimedia-plugin-mediaservice-gstmediaplayer` packages on the emulator. Now I can properly test tablet screen sizes.

I've dropped the whole `<constant> * Theme.pixelRatio`-approach[^1], because the values seemed quite arbitrary and I was entering random numbers. I'm now doing it on multiples of `Theme.itemSizeHuge`, which is easier to reason about. I took the liberty to make some simple other layout changes as well to improve the experience on tablet screens. 

The screen now should look like this: 
![afbeelding](https://github.com/heartfin/harbour-sailfin/assets/4262067/7325f255-81a1-4ff8-986e-8394724b964e)

If you'd like to test, feel free to compile #44 yourself or install the following RPM, which is a compile of commit 60b91335f956d9ab25eef00fc4c8a778c12e7daf: 
[harbour-sailfin-0.5.0-1.aarch64.rpm.gz](https://github.com/heartfin/harbour-sailfin/files/15267035/harbour-sailfin-0.5.0-1.aarch64.rpm.gz)


[^1]: Which I was doing wrong by the way, because I should multiply by the `Theme.pixelRatio`, not divide.
